src/psyclone/domain/nemo/transformations/create_nemo_kernel_trans.py

```diff
@@ -79,7 +80,7 @@ class CreateNemoKernelTrans(Transformation):
                                 ArrayReference[name:'tmp']
                                     Reference[name:'ji']
                                 BinaryOperation[operator:'MUL']
-                                    Literal[value:'2', Scalar<INTEGER, UNDEFINED>]
+                                    Literal[value:'2.0', Scalar<REAL, UNDEFINED>]
```

Comment:
Line > 80 chars

Reply:
There is an issue for this (and for `create_nemo_psy_trans.py`) - see #1528 (where the indentation of the `view()` output will be made controllable (so it can be reduced))